src/main/java/com/anonymous/privacytool/config/SecurityConstants.java

```diff
@@ -0,0 +1,6 @@
+package com.anonymous.privacytool.config;
+
+public class SecurityConstants {
```

Comment:
We should make both configurable in the application config. Will also keep it private.